src/react-apps/ux-editor/src/reducers/formDesignerReducer/formLayoutReducer.ts

```diff
@@ -13,7 +13,8 @@ export interface IFormLayoutState extends IFormDesignerLayout {
 
 const initialState: IFormLayoutState = {
   components: {},
-  order: [],
+  containers: {},
+  order: {},
```

Comment:
Why is `order` now an object and not an array? Considered a 2D array?

Reply:
Since we are adding components/containers to other containers in the order, it's useful to refer to the containers  as keys. 
I did not a consider 2D array, have added an analysis task to issue #66 to determine if that is a better solution.